packages/server/lib/controllers/connection.controller.ts

```diff
@@ -625,14 +624,30 @@ class ConnectionController {
                     }
                 }
 
+                const connCreatedHook = async (res: ConnectionUpsertResponse) => {
+                    void connectionCreatedHook(
+                        {
+                            id: res.id,
+                            connection_id,
+                            provider_config_key,
+                            environment_id: environment.id,
+                            auth_mode: ProviderAuthModes.OAuth1,
+                            operation: res.operation
+                        },
+                        provider,
+                        logContextGetter,
+                        null
+                    );
+                };
+
                 const [imported] = await connectionService.importOAuthConnection(
                     connection_id,
                     provider_config_key,
                     provider,
                     environment.id,
                     account.id,
                     oAuthCredentials,
-                    logContextGetter
+                    connCreatedHook
```

Comment:
as mentioned in the PR description: I didn't want to refactor the entire connection service, so for now a hook function is passed as a parameter to the importConnection methods